icc-x-api/icc-crypto-x-api.ts

```diff
@@ -102,7 +101,9 @@ export class IccCryptoXApi {
   keychainLocalStoreIdPrefix = "org.taktik.icure.ehealth.keychain."
   keychainValidityDateLocalStoreIdPrefix = "org.taktik.icure.ehealth.keychain-date."
   hcpPreferenceKeyEhealthCert = "eHealthCRT"
+  hcpPreferenceKeyEhealthCertEncrypted = "eHealthCRT_encrypted"
   hcpPreferenceKeyEhealthCertDate = "eHealthCRTDate"
+  hcpPreferenceKeyEhealthCertDateEncrypted = "eHealthCRTDate_encrypted"
```

Comment:
Ideally the two other keys should be marked as deprecated with a comment.

Avoid using nonstandard underscores in JSON keys, for consistency reasons.

Reply:
Done.